docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+* If additional look-back time is set for the advanced query rule preview, alerts from source documents that are outside the preview time frame may not appear in the preview ({pull}137422[#137422]).
+* A new Lucene 9 validation change causes errors when creating an event correlation rule for users who upgrade from {stack} version 7.x to 8.x. To fix this, use triple quotes `""" """` for regular expressions within an event correlation rule. 
```

Comment:
@peluja1012 would you mind reviewing/approving this summary - thank you!

Reply:
Hey @nastasha-solomon, I would mention that it only affects users who include regular expressions in their EQL queries. This would make it sound less scary. Other than that, it looks good!